content/tutorials/grafana-fundamentals.md

```diff
@@ -106,216 +119,234 @@ To the far left you can see the _sidebar_, a set of quick access icons for navig
 ```
 
-- In the **Legend** box, enter "{{route}}" to rename the time series in the legend.
-- Click the **General** panel tab, and change the title to "Traffic".
-- Click the arrow in the top-left corner to go back to the dashboard view.
-- Click the **Save dashboard** icon at the top of the dashboard, to save your dashboard.
+- In the **Legend** field, enter `{{route}}` to rename the time series in the legend. The graph legend updates when you click outside the field.
+  Four icons on the left allow you to navigate between the **Queries**, **Visualization**, **General**, and **Alert** tabs. Hover your mouse over the icons to see the name of each tab. 
+- In the **General** tab, change the title to `Traffic`.
+- Click the **Go Back** arrow in the top-left corner to go back to the dashboard view.
+- Click the **Save dashboard** (disk) icon at the top of the dashboard to save your dashboard.
+- Enter a name in the **New name** field and then click **Save**.
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Annotate events" %}}
 
-Whenever things go bad, it can be invaluable to understand the context in which the system failed. Time of last deploy, or database migration can offer insight into what might have caused an outage. Annotations lets you add custom events to your graphs.
+We monitor systems for many reasons, but a common use case is to monitor systems that might go down.
 
-- To manually add an annotation, left-click anywhere in your graph, and click **Add annotation**.
-- In the **Description** box, type "Migrated user database".
-- In the **Tags** box, type "migration".
+Adding annotations to your dashboards is a great way to communicate important events to the rest of your team.
 
-Grafana also lets you annotate a time interval, through _region annotations_.
+When things go bad, it often helps if you understand the context in which the system failed. Time of last deploy, system changes, or database migration can offer insight into what might have caused an outage. Annotations allow you to add custom events to your graphs.
 
-- While pressing Ctrl (or Cmd on macOS), click and drag across the graph using your left mouse button.
-- In the **Description** box, type "Performed load tests".
-- In the **Tags** box, type "testing".
+In the next part of the tutorial, we will simulate some common use cases that someone would add annotations for.
 
-Adding annotations to your dashboards is a great way to communicate important events to the rest of your team.
+- To manually add an annotation, click anywhere in your graph, then click **Add annotation**.
+- In **Description**, enter **Migrated user database**.
+- In **Tags**, enter **migration**.
+- Click **Save**.
+  Grafana adds your annotation to the graph. Hover your mouse over the base of the annotation to read the text.
+
+((Marcus - Since we are having them tag the annotations, you might want to demonstrate how to use them, or at least explain what they do.))
```

Comment:
Are you referring to tags here (as opposed to annotations)?

Reply:
Sorry I was unclear. I meant tags.